Planet Express/src/PlanetExpress.java

```diff
@@ -0,0 +1,158 @@
+import java.util.Scanner;
+
+public class PlanetExpress {
+	private int thirst;
+	private int work;
+	private int horde;
+	private int hunger;
+	private int receipts;
+	private boolean stable;
+	
+
+	public PlanetExpress() {
+		thirst = 50;
```

Comment:
Might be good to use constants for these initial values in case you want to change it somewhere only once